platform/darwin/src/MGLStyle.h

```diff
@@ -194,8 +194,11 @@ MGL_EXPORT
 @property (nonatomic, strong) NS_SET_OF(__kindof MGLSource *) *sources;
 
 /**
- Values describing animated transitions to styling changes, either to the style URL 
- or to individual properties.
+ Values describing animated transitions to styling changes on individual 
+ paint properties. 
+ 
+ @note Transitions between styles changed using `-[MGLMapView setStyleURL:]`
+    are not currently supported.
```

Comment:
In general, we use `@note` for preconditions and postconditions – gotchas of that nature – since the `@pre` and `@post` Doxygen commands aren’t as well-supported. Often it’s sufficient to just have a separate paragraph with no command.

Also, I’d consider the ability to transition between two styles to be a feature in its own right, or at least the lack of this behavior is a bug. Perhaps we don’t need to call out this fact at all in the documentation, unless you think this is a compatibility issue.